src/variant.cpp

```diff
@@ -542,6 +542,31 @@ namespace {
         return v;
     }
 
+    Variant* kono_variant() { //https://en.wikipedia.org/wiki/Five_Field_Kono
+        Variant* v = chess_variant_base()->init();
+        v->maxRank = RANK_5;
+        v->maxFile = FILE_E;
+        v->reset_pieces();
+        v->add_piece(CUSTOM_PIECES, 'p', "mF"); //diagonally, no capture
+        v->startFen = "ppppp/p3p/5/P3P/PPPPP w - - 0 1";
+        v->flagPiece = CUSTOM_PIECES;
+        v->flagRegion[WHITE] = make_bitboard(SQ_A5, SQ_B5, SQ_C5, SQ_D5, SQ_E5, SQ_A4, SQ_E4);
+        v->flagRegion[BLACK] = make_bitboard(SQ_A1, SQ_B1, SQ_C1, SQ_D1, SQ_E1, SQ_A2, SQ_E2);
+        v->numFlag = 7;
+        return v;
+    }
+
+    Variant* fox_and_hounds_variant() { //https://boardgamegeek.com/boardgame/148180/fox-and-hounds
+        Variant* v = chess_variant_base()->init();
+        v->reset_pieces();
+        v->add_piece(CUSTOM_PIECES, 'h', "mfF"); //Hound
+        v->add_piece(CUSTOM_PIECES+1, 'f', "mF"); //Fox
+        v->startFen = "1h1h1h1h/8/8/8/8/8/8/4F3 w - - 0 1";
+        v->stalemateValue = -VALUE_MATE;
+        v->flagRegion[WHITE] = Rank8BB;
```

Comment:
I think the flagPiece is missing here

Reply:
Yes. I also noticed the naming convention for custom pieces had changed so I changed that too.